release.json

```
@@ -7,6 +7,14 @@
 		"JMXFETCH_HASH": "e4cd95f96352b5c010de118f2807011147c3fe605a53f5efed6fbdfda4f6d0f8",
 		"PYTHON_RUNTIMES": "2,3"
 	},
+	"6.13.0-rc.5": {
+		"INTEGRATIONS_CORE_VERSION": "6.13.0-rc.5.1",
```

```suggestion
		"INTEGRATIONS_CORE_VERSION": "6.13.0-rc.5",
```
Last RC was a special case, we had to re-tag integrations-core because another fix was included

---

Actually since we're going to create another tag again for integrations-core, this should be fine 😅

---

Oh missed that with the copy paste, thanks!

---

They should just move the tag.